sendgrid-suppresions.md

```diff
@@ -0,0 +1,51 @@
+## Importing SendGrid Suppression List
+
+ SparkPost CLI can import your SendGrid suppression List(s). However, it has to be in specific format.
+ Follow along to learn how to export suppressions from SendGrid and format them that they're importable by SparkPost.   
+
+### Import Command
+Regardless of suppression type and preparation we need, we will need to run the following command as the final step to import the suppressions to SparkPost.
+
+```
+sp-suppression-list-cli --command sendgrid -f PATH_TO_SENDGRID_BLACKLIST.csv
+```
+Note: Replace `PATH_TO_SENDGRID_BLACKLIST.csv` with correct file that was downloaded in the corresponding step.
```

Comment:
Since this is in a stand-alone document does there need to be any discussion about how to configure API key and/or host?